src/renderer/hooks/useDeeplinking.js

```diff
@@ -53,7 +53,8 @@ export function useDeepLinkHandler() {
     (event: any, deeplink: string) => {
       const { pathname, searchParams } = new URL(deeplink);
       const query = Object.fromEntries(searchParams);
-      const url = pathname.replace(/(^\/+|\/+$)/g, "");
+      const fullUrl = pathname.replace(/(^\/+|\/+$)/g, "");
+      const [url, path] = fullUrl.split("/");
```

Comment:
We did pretty much the same with my PR : 
https://github.com/LedgerHQ/ledger-live-desktop/pull/4148/files#diff-df55ff13a3c744695e8cb8ba3fa2d513e90a10b631704f7be6a3faaffa76247cR57
Risk of merging issues :sweat_smile:

Reply:
Thx will take care of the rebase when yours is merged 👍

Reply:
Thx will take care of the rebase when yours is merged 👍

Reply:
the other pr is merged